libevm/VM.cpp

```diff
@@ -543,6 +544,7 @@ void VM::interpretCases()
 
 		CASE_BEGIN(BALANCE)
 		{
+			m_runGas += toUint64(m_schedule->balanceGas);
```

Comment:
Change it to how it is done for e.g. sha3, i.e. leave it like it is here and change the tier to specialTier.